src/App.tsx

```diff
@@ -1,7 +1,14 @@
 import React from 'react';
+import * as A from './components/atoms';
+import * as E from './containers/enums/icon';
```

Comment:
*idiomatic code*

please follow the best practices for this -  find and share e with us how to import enums - the convention